benchmarks/performance-poetry/complex-poetry/src/main/java/com/squareup/benchmarks/performance/complex/poetry/views/LoaderSpinner.kt

```diff
@@ -0,0 +1,42 @@
+package com.squareup.benchmarks.performance.complex.poetry.views
+
+import android.app.Dialog
+import android.content.Context
+import android.view.ViewGroup.LayoutParams
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.ProgressBar
+import com.squareup.workflow1.ui.AndroidOverlay
+import com.squareup.workflow1.ui.AndroidScreen
+import com.squareup.workflow1.ui.ScreenViewFactory
+import com.squareup.workflow1.ui.ScreenViewHolder
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.container.OverlayDialogFactory
+import kotlin.reflect.KClass
+
+@OptIn(WorkflowUiExperimentalApi::class)
+object LoaderSpinner : AndroidOverlay<LoaderSpinner> {
```

Comment:
@steve-the-edwards This replaces `LoaderContainer`. I'm not thrilled that it requires the author to instantiate the Dialog themselves, where with the old one they could just focus on building a view as usual. 

I'm inclined to follow up by adding a vanilla `Overlay` wrapper type that makes any arbitrary screen a full bleed dialog, but that's too much for a merge PR.